### PR TITLE
[lldb] Add more test cases to TestSwiftNestedGenericClassTest

### PR DESCRIPTION
**Pull request by augusto2112**

### CODE REVIEW

**Comment by augusto2112** [comment]
https://github.com/apple/swift/pull/62854

@swift-ci test

**Comment by augusto2112** [comment]
https://github.com/apple/swift/pull/62854

@swift-ci test

File: lldb/test/API/lang/swift/nested_generic_class/TestSwiftNestedGenericClass.py [hunk 1]
```diff
@@ -5,12 +5,26 @@
 import unittest2
 
 
-class SwiftNestedGenericClassTest(TestBase):
+class TestSwiftNestedGenericClassTest(TestBase):
```
Comment (adrian-prantl) [inline lldb/test/API/lang/swift/nested_generic_class/TestSwiftNestedGenericClass.py]: `TestSwiftNestedGenericClass`?

**Review by adrian-prantl — APPROVED** [review]

**Comment by augusto2112** [comment]
@swift-ci test

File: lldb/test/API/lang/swift/nested_generic_class/TestSwiftNestedGenericClass.py (hunk 1 above)
Comment (kastiglione) [reply lldb/test/API/lang/swift/nested_generic_class/TestSwiftNestedGenericClass.py]: it also doesn't need to match the file name, it can be `TestCase`.

**Pull request merged**